tox.ini

```diff
@@ -27,9 +28,9 @@ skip_install = true
 commands =
     coverage erase
     coverage combine
-    coverage report
     coverage html
     coverage xml
+    coverage report --fail-under=83
```

Comment:
No, please keep it before html/xml to show the results earlier, and maybe add `-m`.

Reply:
Okay. I changed it because the html report was not generated if coverage fell below 83 % and I usually look into the html report as I find it way more readable than the terminal output.

Reply:
You have a very good point there!  I hadn't considered that --fail-under would result in an early exit without HTML reports.